api/oauth.proto

```diff
@@ -78,6 +78,7 @@ message OAuthAccessTokenIdentifiers {
 message OAuthAccessToken {
   UserIdentifiers user_ids = 1 [(validate.rules).message.required = true];
   string user_session_id = 9 [(validate.rules).string.max_len = 64];
+  google.protobuf.Timestamp session_expires_at = 10;
```

Comment:
1. The ordering is a bit strange here. Why not put it together with the other `expires_at`?
2. Please also add a little comment to explain the difference between `expires_at` and `session_expires_at`